apm-sniffer/apm-sdk-plugin/jdbc-commons/src/main/java/org/apache/skywalking/apm/plugin/jdbc/JDBCPluginConfig.java

```diff
@@ -53,6 +60,13 @@
              * Set a negative number to save the complete parameter string to the tag.
              */
             public static int SQL_PARAMETERS_MAX_LENGTH = 512;
+            /**
+             * For the sake of performance, SkyWalking won't save the entire sql body into the tag, but only the first
+             * {@code SQL_BODY_MAX_LENGTH} characters.
+             * <p>
+             * Set a negative number to save the complete sql body to the tag.
+             */
+            public static int SQL_BODY_MAX_LENGTH = 2048;
         }
 
         public static class MARIADB {
```

Comment:
I think we missed an annotation for this, this config would never be initialized. :)

Also, MySQL/POSTGRESQL/MARIADB seems the same, is there any reason to separate them? @ascrutae. I think the only case is, an app is using 2 kinds of DB, but wants to trace one but not for the other. Is this really reasonable?

Reply:
Yes,  you found a :bug: , So can I uniform  MySQL/POSTGRESQL/MARIADB  config to a single one ? 
for blew config 

- TRACE_SQL_PARAMETERS
- SQL_PARAMETERS_MAX_LENGTH
- SQL_BODY_MAX_LENGTH

Reply:
I prefer so, waiting for @ascrutae 's idea.

Reply:
Me too, +1

Reply:
@xbkaishui Seems these configs could be merged.

Reply:
Ok, will change

Reply:
done